### PR TITLE
QF-1644 Add OfS shutter pages and single question

### PR DESCRIPTION
**Pull request by SawNawDfE**

### CODE REVIEW

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.AssessorService.Web/Helpers/OfsShutterPageSelection.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿namespace SFA.DAS.AssessorService.Web.Helpers
+{
+    public class OfsShutterPageSelection
```
[the remark this answers is not in the document]
Comment (SawNawDfE) [reply src/SFA.DAS.AssessorService.Web/Helpers/OfsShutterPageSelection.cs:3]: I do, ASP.NET did not...

File: src/SFA.DAS.AssessorService.Database/Tables/Standards.sql [hunk 2]
```diff
@@ -31,6 +31,7 @@
 	[EqaProviderContactEmail] VARCHAR(500) NULL,
 	[OverviewOfRole] VARCHAR(500),
 	[CoronationEmblem] BIT NOT NULL DEFAULT 0,
+	[EpaoMustBeApprovedByRegulatorBody] BIT NOT NULL Default 0,
```
[the remark this answers is not in the document]
Comment (SawNawDfE) [reply src/SFA.DAS.AssessorService.Database/Tables/Standards.sql]: Sorted

File: src/SFA.DAS.AssessorService.Web/Views/Standard/ApplyStandardConfirmOfs.cshtml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    Layout = "_Layout";
+}
+
+<a asp-controller="Dashboard" asp-action="Index" class="govuk-back-link">Back to dashboard</a>
+
+<main class="govuk-main-wrapper " id="main-content" role="main">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            @if (Model.ShowNeedToRegisterPage)
```
[the remark this answers is not in the document]
Comment (SawNawDfE) [reply src/SFA.DAS.AssessorService.Web/Views/Standard/ApplyStandardConfirmOfs.cshtml]: Deleted

File: src/SFA.DAS.AssessorService.Web/Views/Standard/ApplyStandardConfirmOfs.cshtml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+@using SFA.DAS.AssessorService.Web.Controllers.Apply
+@using SFA.DAS.AssessorService.Web.ViewModels.Standard
+@model ApplyStandardConfirmOfsViewModel
+
+@{
+    ViewBag.Title = "OfS Shutter Page";
+    Layout = "_Layout";
+}
+
+<a asp-controller="Dashboard" asp-action="Index" class="govuk-back-link">Back to dashboard</a>
```
[the remark this answers is not in the document]
Comment (SawNawDfE) [reply src/SFA.DAS.AssessorService.Web/Views/Standard/ApplyStandardConfirmOfs.cshtml]: Fixed

**Review by ChrisJWoodcock — APPROVED** [review]

**Pull request merged**